WordPress/src/main/res/layout/login_magic_link_sent_screen.xml

```diff
@@ -44,8 +44,9 @@
         style="@style/Base.TextAppearance.AppCompat.Caption"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_below="@+id/login_open_email_client"
+        android:layout_alignParentBottom="true"
         android:layout_marginTop="@dimen/margin_extra_extra_extra_large"
+        android:layout_marginBottom="@dimen/margin_small_medium"
```

Comment:
OK, it seems we missed using the proper class and style for this secondary button so, if you don't mind, let's copy the corresponding button style from the `login_magic_link_request_screen`, shall we? It will not need this extra bottom margin then. Thanks!